beta/aws-ecsfargate-cfn/README.md

```diff
@@ -146,87 +145,80 @@ Edit the example above as follows before running the command from the working di
 
 - Uncomment the appropriate lines as needed to adjust the CIDR block for the VPC, the scale of the deployment based on its provisoning volume, and add tags as key-value pairs to apply to all supported resources in the stack.
 - If you prefer, set the value for `--stack-name` to choose your own (for example, `--stack-name your-stack-name`). CloudFormation will use the stack name (or a truncated version where needed) as a prefix when naming the created AWS resources.
-  > **Note**
-  >
-  > 📄 We use `op-scim-bridge` as a shorthand for 1Password SCIM Bridge throughout our examples and supporting documentation, but the choice is arbitrary.
+> [!NOTE]
+> 📄 We use `op-scim-bridge` as a shorthand for 1Password SCIM Bridge throughout our examples and supporting documentation, but the choice is arbitrary.
```

Comment:
Hmm, do we need the emoji here still? GitHub styles these nicely so I feel like it's redundant to also use emoji.

```suggestion
> We use `op-scim-bridge` as a shorthand for 1Password SCIM Bridge throughout our examples and supporting documentation, but the choice is arbitrary.
```

Reply:
Good catch. I removed some others, but missed this one. 😅